docs/README.md

```diff
@@ -0,0 +1,5 @@
+Pilosa docs are maintained here, to stay in sync with the codebase.
+
+Please visit [our website](https://www.pilosa.com/docs) to view the docs complete with styles, diagrams, and comprehensive search.
```

Comment:
Always end pilosa.com urls with a slash, as they always get redirected to include a slash.